AndelaOSP/wire
# #152107394 Integrate Redux

#### What does this PR do?
Integrate Redux to project

#### Description of Task to be completed?
Integrate Redux to project

#### How should this be manually tested?

- Clone repo: git clone https://github.com/AndelaOSP/wire.git
- Install packages: yarn install
- Run the project: yarn start
- Access the project: http://localhost:8080/
- Create an `action, related reducer and fire up the action`
- Open dev tools on your browser 
- Under Redux, you should see the action fired and the state of the application store.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[152107394](https://www.pivotaltracker.com/story/show/152107394)

#### Screenshots (if appropriate)
<img width="1280" alt="screen shot 2017-11-06 at 13 31 10" src="https://user-images.githubusercontent.com/25595450/32437322-fbb6de48-c2f7-11e7-923c-c51152fd6a3a.png">

#### Questions:
N/A
